readme.md

```diff
@@ -44,25 +44,120 @@ A helper function for constructing native Pact commands.
 ```
 Pact.lang.mkExp(<string>, *args) -> <string>
   ex: mkExp("todos.edit-todo", 1, "bar") -> '(todos.edit-todo 1 "bar")'
+
+Pact.lang.mkMeta(<sender:string> , <chainId:string>, <gasPrice: nunmber>, <gasLimit: number>) -> <meta: object>
+  ex: mkMeta("Bob", "Chain-1", 20, 30) -> { "sender": "Bob", "ChainId": "Chain-1", "gasPrice": 20, "gasLimit": 30 }
 ```
 
 NB: `JSON.stringify`, which is used here, generally converts floating point numbers correctly but fails for high precision scientific numbers < 0; you will need to manually convert them.
 e.g. `JSON.stringify(.0000001) -> '1e-7'` is incorrect as Pact has infinite precision decimals but does not interpret scientific numbers, so the proper conversion would be `JSON.stringify(.0000001) -> '0.0000001'`
 
 
-### Simple API
+
+### Simple API Fetch
+
+Simple fetch functions to make API request to a running Pact Server and retrieve the results.
+
+```
+* A Command Object to Execute in send or local.
+* @typedef {Object} execCmd
+* @property pactCode {string} - pact code to execute
+* @property keyPairs {array or object} - array or single ED25519 keypair
+* @property nonce {string} - nonce value, default at current time
+* @property envData {object} - JSON message data for command, default at empty obj
+* @property meta {object} - meta information, see mkMeta
+*/
+```
+```
+## Make API request to execute a command or commands in the public server and retrieve request keys of the txs.
+
+Pact.fetch.send([<execCmd:object>], <apiHost:string>) -> {"requestKeys": [...]}
+
+  ex:
+    const cmds = [{
+                    keyPairs: KEY_PAIR,
+                    pactCode: "todos.delete-todos "id-1""
```

Comment:
these need escaped quotes or single quotes around the whole thing

Reply:
Changed to Single quotes